google/cloud/vision/v1/vision.yaml

```diff
@@ -0,0 +1,27 @@
+type: google.api.Service
+config_version: 2
+name: vision.googleapis.com
+title: Google Cloud Vision API
+
+apis:
+- name: google.cloud.vision.v1.ImageAnnotator
+
+documentation:
+  summary:
+    'Integrates Google Vision features, including image labeling, face, logo,
+    and landmark detection, optical character recognition (OCR), and detection
+    of explicit content, into applications.'
+
+authentication:
+  rules:
+    - selector: '*'
+      allow_without_credential: true
```

Comment:
Hm... Removed.

I actually brought this line from internal, but at least cloud-vision API site says about the authentication (https://cloud.google.com/vision/docs/auth-template/cloud-api-auth), so this is probably not the case of this API at least for now.

But I think gapic-generated code does not consider APIs which don't require credentials, so this is an interesting topic.